modules/@apostrophecms/module/index.js

```diff
@@ -177,6 +177,11 @@ module.exports = {
           return async function(req, res) {
             try {
               const result = await fn(req);
+
+              if (req.method === 'GET' && req.user) {
+                res.header('Cache-Control', 'no-store');
```

Comment:
I first re-used the `setMaxAge` function, but that was too much of a refactoring to fit in this case, so I decided to keep it simple (and quick), by only setting the Cache-Control directly.

But if we want the session to be checked as well, as it is done in `setMaxAge`, then `setMaxAge` will have to be re-adapted.

@boutell WDYT?

Reply:
It's OK to check just `req.user` for this.

Reply:
tests for this added in f9566b2ff